python/.env.example

```diff
@@ -1,10 +1,15 @@
 OPENAI_API_KEY=""
+OPEN_AI_CHAT_MODEL_ID=""
+OPEN_AI_TEXT_MODEL_ID=""
+OPEN_AI_EMBEDDING_MODEL_ID=""
 OPENAI_ORG_ID=""
-AZURE_OPENAI_SYSTEM_MESSAGE="You are an AI assistant that helps people find information"
-AZURE_OPENAI_API_VERSION="2024-02-15-preview"
-AZURE_OPENAI_DEPLOYMENT_NAME=""
+AZURE_OPENAI_CHAT_DEPLOYMENT_NAME=""
+AZURE_OPENAI_TEXT_DEPLOYMENT_NAME=""
+AZURE_OPENAI_EMBEDDING_DEPLOYMENT_NAME=""
 AZURE_OPENAI_ENDPOINT=""
 AZURE_OPENAI_API_KEY=""
+AZURE_OPENAI_SYSTEM_MESSAGE="You are an AI assistant that helps people find information"
```

Comment:
do we actually read this one from our code? if not let's either implement that, or delete from this example!